---
bool success
static_map/Robot robot_shape
static_map/MapObject terrain_shape
static_map/MapLayer[] terrain_layers
